release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -175,6 +175,52 @@ With this release, you can access the NMstate Operator and resources such as the
 [id="ocp-4-14-storage"]
 === Storage
 
+[id="ocp-4-14-storage-standardized-sts-config"]
+==== Standardized STS configuration workflow
+{product-title} 4.14 provides a streamlined and standardized procedure to configure Security Token Service (STS) with the AWS Elastic File Storage (EFS) Container Storage Interface (CSI) Driver Operator.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc#efs-sts_persistent-storage-csi-aws-efs[Configuring AWS EFS CSI Driver Operator with Security Token Service].
+
+[id="ocp-4-14-storage-rwod-access-mode"]
+==== Read Write Only Pod access mode (Technology Preview)
+{product-title} 4.14 introduces a new access mode for persistent volumes (PVs) and persistent volume claims (PVCs) called ReadWriteOncePod (RWOD), which can be used only in a single pod on a single node. This is compared to the existing ReadWriteOnce access mode where a PV or PVC can be used on a single node by many pods. This feature is supported with Technology Preview status.
+
+For more information, see xref:../storage/understanding-persistent-storage.adoc#pv-access-modes_understanding-persistent-storage[Access modes].
+
+[id="ocp-4-14-storage-robust-volmgr"]
+==== Robust VolumeManager reconstruction (Technology Preview)
```

Comment:
As per the last conversation, this story was moved to 4.15, in that case can we please remove documentation for this story? @jsafrane Can you please confirm?

Reply:
Done.